frontend/src/AddQuestionDialog.jsx

```diff
@@ -1,170 +1,179 @@
-import { useState, useRef, useEffect } from 'react';
+import { useState, useRef, useEffect } from "react";
 import {
-	Button,
-	TextField,
-	Dialog,
-	DialogActions,
-	DialogContent,
-	DialogContentText,
-	DialogTitle,
-	Stack,
-	MenuItem,
-	Box,
-} from '@mui/material';
-import CustomSnackbar from './CustomSnackbar';
+  Button,
+  TextField,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+  Stack,
+  MenuItem,
+  Box,
+} from "@mui/material";
+import CustomSnackbar from "./CustomSnackbar";
+import ReactQuill from "react-quill";
```

Comment:
i'm getting this error when i tried to run the localhost. i think we have to run the command `npm install react-quill --save` inside the 'frontend' folder to install the package locally.

<img width="893" alt="Screenshot 2023-10-04 at 6 25 37 PM" src="https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g49/assets/77200805/5d8c12a2-d975-4da2-b77f-cc09c69cd9ba">

Reply:
Sorry, I installed the package outside the frontend folder. It is now included, thanks for reminding.